include/nbl/builtin/hlsl/bxdf/fresnel.hlsl

```diff
@@ -472,7 +455,7 @@ struct Dielectric
         return __call(orientedEta2, clampedCosTheta);
     }
 
-    OrientedEtas<T> getRefractionOrientedEta() NBL_CONST_MEMBER_FUNC { return orientedEta; }
+    scalar_type getRefractionOrientedEta() NBL_CONST_MEMBER_FUNC { return orientedEta.value[0]; }   // expect T to be monochrome?
```

Comment:
its a bit of a complicated topic, its possible to have RGB fresnel without dispersion by fixing the refraction Eta to be something else than the etas used to compute RGB reflectance or some sort of interpolation of them.

This is a good default cause the Etas wont differ that much anyway